src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsEnforcementSpec.groovy

```diff
@@ -473,4 +477,29 @@ class PriceFloorsEnforcementSpec extends PriceFloorsBaseSpec {
         assert response.seatbid?.first()?.bid?.collect { it.id } == [bidResponse.seatbid.first().bid.first().id]
         assert response.seatbid.first().bid.collect { it.price } == [floorValue]
     }
+
+    def "PBS floor enforcement shouldn't fail when bidder request is null"() {
```

Comment:
`null` is a term from unit tests. Functionality-wise, we are talking about a case when no bidder call was made at all.
`"PBS floor enforcement shouldn't fail when bidder request was not made"`